src/arrayMethodSort.js

```diff
@@ -4,9 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const tringCompare = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA === stringB) {
+      return 0;
+    }
+
+    return stringA > stringB ? 1 : -1;
+  };
+
+  [].__proto__.sort2 = function(compareFunction = tringCompare) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = stringCompare) {
```

typo in word stringCompare